jenkins/image_building/build-image.sh

```diff
@@ -3,15 +3,21 @@
 set -eux
 
 current_dir="$(dirname "$(readlink -f "${0}")")"
+REPO_ROOT="$(realpath "${current_dir}/../..")"
 
 cleanup() {
   deactivate || true
+  cd "${REPO_ROOT}" || true
   sudo rm -f "${img_name}".{raw,qcow2}
+  sudo rm -rf "${img_name}.d"
   sudo rm -rf "${current_dir}/dib"
 }
 
 trap cleanup EXIT
 
+# Make sure we run everything in the repo root
+cd "${REPO_ROOT}" || true
```

Comment:
cd is fine here you are not expecting to leave this location.

Reply:
Thanks. But I still would like to keep this!

One major problem with this script (and with many of our scripts), is that we always assume that the script starts from some location, in this case the REPO_ROOT. It works fine for CI, but in case one wants to run it from somewhere else (like cd jenkins/image_building && ./build-image.sh), it wouldn't work. This cd will help solve that issue, and it's harmless otherwise.

Reply:
Yes I agree that is why I said it is fine :) .